core/controllers/review_tests.py

```diff
@@ -90,11 +91,19 @@ def get(self, story_id):
         if len(latest_completed_node_ids) == 0:
             raise self.PageNotFoundException
 
-        acquired_skill_ids = story.get_acquired_skill_ids_for_node_ids(
-            latest_completed_node_ids
-        )
+        try:
+            skills = skill_services.get_multi_skills(
+                story.get_acquired_skill_ids_for_node_ids(
+                    latest_completed_node_ids
+                ))
+        except Exception, e:
+            raise self.PageNotFoundException(e)
+        skills_with_description = {}
+        for skill in skills:
+            skills_with_description[skill.id] = skill.description
```

Comment:
Following up with the naming comment in the frontend code.... This variable seems incorrectly named then. It is not a skill with description but just the skill descriptions, right?

Reply:
Makes sense, I have changed it in both practice sessions and review tests!